assets/embedded-files/tekton/pipeline.yaml

```diff
@@ -79,6 +79,21 @@ spec:
     workspaces:
     - name: source
       workspace: source
+  - name: clean
+    taskRef:
+      name: clean
+    params:
+      - name: APP_NAME
+        value: "$(params.APP_NAME)"
+      - name: ORG
+        value: "$(params.ORG)"
+      - name: STAGE_ID
+        value: "$(params.STAGE_ID)"
+    runAfter:
+    - stage
```

Comment:
Maybe it's better to run the cleanup after `run` instead of `stage`. `run` is the task that creates/updates the Deployment Service and Ingress, so the staging process is really done when we finally run the provided image.

Reply:
Can be done. Trivial one-line change. Just a moment.

Reply:
Done, commit 3c392323b8a6f3030e6da07f00aab5e3cc526510